joint_state_publisher_gui/src/joint_state_publisher_gui/__init__.py

```diff
@@ -0,0 +1,196 @@
+import math
```

Comment:
Nit, missing license boilerplate comment

Reply:
Fixed in 12bd0e5.